### ISSUE TITLE
Anti-Beta-2-Glycoprotein I IgM antibody positivity

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7060]
**Preferred term label:**
Anti-Beta-2-Glycoprotein I IgM antibody positivity

**Synonyms**
aβ2-GPI IgM antibody positivity
Anti-Beta2GPI IgM antibody positivity

**Definition (free text, please give PubMed ID)**
PMID: 34218350, PMID: 33957795

**Parent term (use hpo.jax.org/app)**
HP:0032376

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:80, ORPHA:464343

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
added
